src/Hooks/SiteEnvLookup.php

```diff
@@ -118,12 +121,20 @@ protected function determineSite()
         // if present.
         $site = $this->getConfig()->get('site');
         if (!empty($site)) {
+            $this->logger->info(
+                sprintf('Missing "site" argument. Setting to "%s" (from TERMINUS_SITE env var).', $site)
```

Comment:
Useful, but I am not sure if we do or do not want to add output here. Could interfere with scripts.

If we do keep this,`logger` has the built-in ability to do variable insertion:
```
$this->logger->info('Missing "site" argument. Setting to "{site}" (from TERMINUS_SITE env var).', ['site' => $site]);
```

Reply:
This output is visible with `-v` option only